docs/getting-started.md

```diff
@@ -62,8 +67,63 @@ Under most circumstances you won't need to run the application in Docker locally
 - copy the `.env.example` file, save as `.env` and populate the secrets within
 - run `docker-compose up`
 
+## Run tests
+
+### Unit tests
+
+To run the project's unit tests, open a terminal in the project directory and run:
+
+```bash
+cd tests/DfE.FindInformationAcademiesTrusts.UnitTests
+dotnet test
+```
+
+### Accessibility and UI tests
+
+Accessibility and UI tests are written using [Playwright](https://playwright.dev/), with external dependencies (e.g. APIs) mocked using [WireMock](https://github.com/HBOCodeLabs/wiremock-captain). 
+To run these tests locally it is easiest to run your app and the mock API using Docker:
+
+```bash
+cd DfE.FindInformationAcademiesTrusts.UnitTests/playwright
+
+# run docker image
+docker compose -f ../../docker/docker-compose.ci.yml up -d
```

Comment:
pop a --build on the end of this command to force the image to rebuild. It takes a while but guarantees that the latest code is in there.